Jenkinsfile

```diff
@@ -0,0 +1,50 @@
+/**
+* Android Jenkinsfile
+*/
+node("android"){
+  stage("Checkout"){
+    checkout scm
+  }
+
+  stage ("Prepare"){
+    if(params.FH_CONFIG_CONTENT) {
```

Comment:
We can remove the FH_CONFIG thing and move the command `sh 'chmod +x ./gradlew'` from the build stage to this one.